src/Model/Channel.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2023, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+ namespace Friendica\Model;
+
+use Friendica\Model\User;
+use Friendica\Content\Entity\Conversation\Channel as ChannelEntity;
+use Friendica\Core\L10n;
+use Friendica\Database\Database;
+use Psr\Log\LoggerInterface;
+
+final class Channel extends \Friendica\BaseModel
```

Comment:
Since you're calling the constructor of the `ChannelTab` entity, this should be a factory in `Content/Conversation/Factory/ChannelTab.php`.

Reply:
I tried using "BaseFactory", but that seem to need some additional class that I don't have, since that is meant for this whole table management.

Reply:
`BaseFactory` only has a `Logger` dependency, do you mean the `ICanCreateFromTableRow` interface? You can add it later, when there's an actual table for this.